lstchain/mc/sensitivity.py

```diff
@@ -71,40 +72,45 @@ def process_mc(dl1_file, dl2_file, mc_type):
     sim_par = read_sim_par(dl1_file)
     events = pd.read_hdf(dl2_file)
 
-    #Filters:
-
+    # Filters:
+    # TO DO: These cuts must be given in a configuration file
+    # By now: only cut in leakage and intensity
+    # we use all telescopes (number of events needs to be multiplied 
+    # by the number of LSTs in the simulation)
     filter_good_events =  (events.leakage < 0.2) & \
-                          (events.intensity > np.log10(200)) & \
-                          (events.wl > 0.1) & \
-                          (events.tel_id==1)
+                          (events.intensity > np.log10(50))
+                          #(events.tel_id==1)
+                          #(events.wl > 0.1) 
+    
 
     events = events[filter_good_events]
 
-    e_reco = 10**events.mc_energy.to_numpy() * u.GeV
+    e_reco = 10**events.reco_energy.to_numpy() * u.GeV
     e_true = 10**events.mc_energy.to_numpy() * u.GeV
```

Comment:
those values are also in linear scale?

Reply:
is this a question? If I'm not wrong, those energy values were in log-scale

Reply:
Same thing here, in the files we have log_mc_energy in log scale, and mc_energy, and log_reco_energy and reco_energy

Reply:
Many things changed since the opening of this PR... corrected also!